paramiko/hostkeys.py

```diff
@@ -25,11 +25,11 @@
 
 from paramiko.py3compat import b, u, encodebytes, decodebytes
 
-try:
-    from collections import MutableMapping
-except ImportError:
+try:                                                    # pragma: no cover
+    from collections import MutableMapping              # pragma: no cover
+except ImportError:                                     # pragma: no cover
     # noinspection PyUnresolvedReferences
-    from UserDict import DictMixin as MutableMapping
+    from UserDict import DictMixin as MutableMapping    # pragma: no cover
```

Comment:
It's worth it when I try to develop and uses hotkeys to jump to next covered part :) But they can be removed if you wishes.